calendar-cli.py

```diff
@@ -139,8 +143,14 @@ def _calendar_addics(caldav_conn, ics, uid, args):
         raise ValueError("Nothing to do/invalid option combination for 'calendar add'-mode; either both --icalendar and --nocaldav should be set, or none of them")
         return
 
-    c = find_calendar(caldav_conn, args)
-    c.add_event(ics)
+    try:
+        c = find_calendar(caldav_conn, args)
+        c.add_event(ics)
+    except:
+        if "200 OK" in str(e):
```

Comment:
Can we be sure that an exception containing the string "200 OK" means a duplicate?  I'm all for presenting errors and warnings in a nicer way than a traceback, but it's better with a traceback than a potentially misleading error message.